examples/AspNetCore/appsettings.json

```diff
@@ -17,9 +17,6 @@
     "Endpoint": "http://localhost:14268",
     "Protocol": "UdpCompactThrift"
   },
-  "Prometheus": {
```

Comment:
was not using so removed. (we can add back if we want to demonstrate how to configure things from appsettings.json in the future.)